.github/workflows/test-package.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
           pip install ".[test]"
       - name: lint python
         run: |
-          ruff check
+          ruff check --fix
```

Comment:
Why this change? --fix will _Apply fixes to resolve lint violations._, we probably don't want to do this in our CI

Reply:
I'm not sure what's going on with ruff. It errors out without this, but locally everything passes the ruff check.

Reply:
I just guessed that it would require an empty line after the internal packages and it worked lol